Shared/RollingFileLogger.swift

```diff
@@ -27,6 +27,27 @@ open class RollingFileLogger: XCGLogger {
         self.sizeLimit = sizeLimit
         self.logDirectoryPath = logDirectoryPath
         super.init()
+        
+        if !AppConstants.BuildChannel.isRelease {
```

Comment:
This is only really useful for builds made on the machine right? Is there anyway to guarantee this doesn't get called for any build on a dev machine and not to do with release configuration?

Reply:
This should work on beta builds too, flags for release and beta are different

So it should work on dev and beta, not on release